src/phases/execution/MultiDeletes.yml

```diff
@@ -0,0 +1,163 @@
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: Populate the collection for mass deletion.
+- Name: PopulateInitialData0
+  Type: MonotonicSingleLoader
+  Threads: 100
+  Phases:
+  - *Nop
+  - Repeat: 1
+    MetricsName: PrePopulation0
+    Database: *Database
+    Collection: *MassDeletionsColl
+    DocumentCount: *DocumentCount
+    BatchSize: 1000
+    Document:
+      padding: {^FastRandomString: {length: *DocumentSizePaddingBytes}}
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: Gaurd against timout due to no logging during PrePoulation0.
+- Name: LoggingActor0
+  Type: LoggingActor
+  Threads: 1
+  Phases:
+    - *Nop
+    - LogEvery: 9 minutes
+      Blocking: None
+    - *Nop
+    - *Nop
+    - *Nop
+
+# Phase 3: Issue a large delete that should take a considerable amount of time to run. This delete
+# will run with no concurrent writes to the database.
+- Name: LargeMultiDelete0
+  Type: CrudActor
+  Database: *Database
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    MetricsName: DuringMassDeletion0
+    Collection: *MassDeletionsColl
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: deleteMany
+          Filter: {_id: {$gte: 0}}
+          OperationOptions:
+            WriteConcern: { Level: "majority" }
+  - *Nop
+  - *Nop
+
+# Phase 4: Populate the collection for mass deletion again. This time concurrent writes will be
+# issued to another collection in the same database.
+# NOTE: The MonotonicSingleLoader counter will start where it left off in the first instance of this
+# Actor in a previous phase.
+- Name: PopulateInitialData1
+  Type: MonotonicSingleLoader
+  Threads: 100
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    MetricsName: PrePopulation1
+    Database: *Database
+    Collection: *MassDeletionsColl
+    DocumentCount: *DocumentCount
+    BatchSize: 1000
+    Document:
+      padding: {^FastRandomString: {length: *DocumentSizePaddingBytes}}
+  - *Nop
+
+# Phase 4: Gaurd against timout due to no logging during PrePoulation1.
+- Name: LoggingActor1
+  Type: LoggingActor
+  Threads: 1
+  Phases:
+    - *Nop
+    - *Nop
+    - *Nop
+    - LogEvery: 9 minutes
+      Blocking: None
+    - *Nop
+
+# Phase 5: Issue a large delete that should take a considerable amount of time to run. This delete
+# will run with concurrent writes to the database.
+- Name: LargeMultiDelete1
+  Type: CrudActor
+  Database: *Database
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    MetricsName: DuringMassDeletion1
+    Collection: *MassDeletionsColl
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: deleteMany
+          Filter: {_id: {$gte: 0}}
+          OperationOptions:
+            WriteConcern: { Level: "majority" }
+
+# Phase 5: Perform some write operations - see how write latency is impacted by the large delete.
+- Name: Insert
```

Comment:
We can create a ticket to make this more robust --- with other CRUD operations, just trying to get vanilla implementation out right now